### PR TITLE
[AST Builder] Implement Lpad, Rpad Funciton

### PR DESCRIPTION
**Pull request by seonghun-dev**
Resolve : https://github.com/gluesql/gluesql/issues/751

### CODE REVIEW

File: core/src/ast_builder/expr/function.rs [hunk 1]
```diff
@@ -37,6 +37,8 @@ pub enum FunctionNode {
     Lcm(ExprNode, ExprNode),
     GenerateUuid,
     Repeat(ExprNode, ExprNode),
+    Lpad(ExprNode, ExprNode, Option<ExprNode>),
```
Comment (ever0de) [inline core/src/ast_builder/expr/function.rs:40]: How about going with the old structure? (in core/src/ast/funtion.rs)

```suggestion
    Lpad {
        expr: Expr,
        size: Expr,
        fill: Option<Expr>,
    }
```

File: core/src/ast_builder/expr/function.rs [hunk 2]
```diff
@@ -107,6 +109,22 @@ impl TryFrom<FunctionNode> for Function {
             FunctionNode::Repeat(expr, num) => expr
                 .try_into()
                 .and_then(|expr| num.try_into().map(|num| Function::Repeat { expr, num })),
+            FunctionNode::Lpad(expr_node, size_node, fill_node) => {
+                let fill = fill_node.map(|fill| fill.try_into().unwrap());
```
Comment (ever0de) [inline core/src/ast_builder/expr/function.rs:113]: Requires error handling for `unwrap`.

**Review by ever0de — APPROVED** [review]
A small comment about refactoring

**Review by panarch — APPROVED** [review]
All looks good! Thanks a lot 👍 
also thanks for the review @ever0de

**Pull request merged**